Added Issue template

Konijima opened the pull request.

Konijima committed:
Added Issue and Feature request template

Shurutsue reviewed (dismissed):
OS example should rather be like Windows/linux.
Device/Browser seem relatively unnecessary in this regard.
Possibly a field for the error messages/log.
(Also seems to be missing feature request template)

Konijima commented:
> OS example should rather be like Windows/linux. Device/Browser seem relatively unnecessary in this regard. Possibly a field for the error messages/log. (Also seems to be missing feature request template)

I went a little too fast, will fix and add the feature request as it seems to be missing in this commit.

Chuckleberry-Finn commented:
Console.txt provides a crap ton of info on the machine. We should probably insist they just provide one anyway as it may shed alot more light to their issue then they can.

Also, humble flex example: 
```
LOG  : General     , 1636693924874> ===== System specs =====
LOG  : General     , 1636693924876> OS: Windows 10, version: 10.0, arch: amd64
LOG  : General     , 1636693924876> Processor: Intel64 Family 6 Model 60 Stepping 3, GenuineIntel
LOG  : General     , 1636693924877> Processor cores: 8
LOG  : General     , 1636693924878> Available processors (cores): 8
LOG  : General     , 1636693924880> Memory free: 1706.0 MB
LOG  : General     , 1636693924880> Memory max: 2048.0 MB
LOG  : General     , 1636693924881> Memory  total available to JVM: 1800.0 MB
LOG  : General     , 1636693924901> C:\, Total: 464.54816 GB, Free: 96.43858 GB
LOG  : General     , 1636693924902> D:\, Total: 931.51074 GB, Free: 482.3924 GB
LOG  : General     , 1636693924903> I:\, Total: 465.7588 GB, Free: 459.6719 GB
LOG  : General     , 1636693925147> Mobo = [Product=Z97-A]
LOG  : General     , 1636693925424> CPU = [Manufacturer=GenuineIntel,MaxClockSpeed=4001,Name=Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz]
LOG  : General     , 1636693925705> Graphics = [AdapterRAM=4293918720,DriverVersion=30.0.14.7212,Name=NVIDIA GeForce RTX 3060 Ti]
LOG  : General     , 1636693925950> VideoMode = [VideoModeDescription=1920 x 1080 x 4294967296 colors]
LOG  : General     , 1636693926185> Sound = { [Manufacturer=(Generic USB Audio),Name=USB Audio Device], [Manufacturer=Realtek,Name=Realtek High Definition Audio], [Manufacturer=NVIDIA,Name=NVIDIA Virtual Audio Device (Wave Extensible) (WDM)], [Manufacturer=NVIDIA,Name=NVIDIA High Definition Audio] }
LOG  : General     , 1636693926417> Memory RAM = { [Capacity=8589934592,Manufacturer=0420], [Capacity=8589934592,Manufacturer=0420], [Capacity=8589934592,Manufacturer=0420], [Capacity=8589934592,Manufacturer=0420] }
LOG  : General     , 1636693926418> ========================
```

Konijima committed:
Update bug_report.md

Konijima committed:
Update bug_report.md

Konijima committed:
Update bug_report.md

Shurutsue committed:
Update bug_report.md

Added simple copy/paste for the address bar for windows to get to the location of the console.txt

Chuckleberry-Finn commented on .github/ISSUE_TEMPLATE/bug_report.md:
````diff
@@ -1,46 +1,34 @@
+On Windows found at : `C:\Users\<Username>\Zomboid\console.txt`
+
+To quickly go to that location, you can copy `%UserProfile%\Zomboid\` into the address bar on the top of the file explorer.
+```
````
It should be noted this log resets when the game is started up, so the log has to be retrieved after the error occurs but before the game is restarted.

Chuckleberry-Finn committed:
Update bug_report.md

Chuckleberry-Finn committed:
Update bug_report.md

Chuckleberry-Finn committed:
Update bug_report.md
